vm/systemSmartContracts/auction.go

```diff
@@ -903,6 +967,8 @@ func (s *stakingAuctionSC) claim(args *vmcommon.ContractCallInput) vmcommon.Retu
 		log.Debug("transfer error on finalizeUnStake function",
 			"error", err.Error(),
 		)
+
```

Comment:
delete empty line - log debugs might be deleted

Reply:
done.